src/bindings/js/node/src/model_wrap.cpp

```diff
@@ -154,3 +156,25 @@ Napi::Value ModelWrap::get_friendly_name(const Napi::CallbackInfo& info) {
     const auto friendly_name = _model->get_friendly_name();
     return Napi::String::New(env, friendly_name);
 }
+
+Napi::Value ModelWrap::get_output_shape(const Napi::CallbackInfo& info) {
+    if (info.Length() != 1 || !info[0].IsNumber()) {
+        reportError(info.Env(), "Invalid argument. Expected a single number.");
+        return info.Env().Undefined();
+    }
+
+    auto idx = info[0].As<Napi::Number>().Int32Value();
+    auto cm_outputs = _model->outputs();  // Output<Node>
+
+    if (idx >= 0 && idx < cm_outputs.size()) {
+        try {
+            return cpp_to_js<ov::Shape, Napi::Array>(info, cm_outputs[idx].get_shape());
+        } catch (const std::exception& e) {
+            reportError(info.Env(),e.what());
+            return info.Env().Undefined();
+        }
+    } else {
+        reportError(info.Env(), "Invalid output index.");
+        return info.Env().Undefined();
+    }
```

Comment:
I propose to do not process index checking manually

```suggestion
    try {
      auto idx = info[0].As<Napi::Number>().Int32Value();
      auto output = _model->output(idx);

      return cpp_to_js<ov::Shape, Napi::Array>(info, output.get_shape());
    } catch (const std::exception& e) {
          reportError(info.Env(),e.what());
          return info.Env().Undefined();
    }
```

Reply:
hey @vishniakov-nikolai  for this then for the following test case it throws a weird actual error so should I modify expected error according to that
![image](https://github.com/openvinotoolkit/openvino/assets/116452077/e29e4e0e-b51e-4c08-bc50-5b2c2ed5f479)

Reply:
Yes please replace error message in tests by that